quickstart.rb

```diff
@@ -20,15 +21,15 @@ class Quickstart < Formula
   end
 
   def install
+    FileUtils.mkdir_p(Dir.home + "/.config/kn/plugins")
     if OS.mac?
-      FileUtils.mv("kn-quickstart-darwin-amd64", "kn-quickstart")
+      FileUtils.mv("kn-quickstart-darwin-amd64", Dir.home + "/.config/kn/plugins/kn-quickstart")
```

Comment:
We've debated this... on the one hand installing in the normal location by brew would allow easy clean up. But that requires users to specify lookup in path for `kn`. See my [comment]() below.

Ideally a symlink would be better since that would allow both `PATH` and `lookup-path` to work